data/mock_datasets.csv

```diff
@@ -0,0 +1,149 @@
+query,sample
```

Comment:
awesome!

but suggest using relative paths in this file - as it is you'll have to have the exact disk/volume setup specified as here to run this CSV.

Reply:
Changed the mock_datasets.csv to relative paths :)